tests/NUnit.OneTimeSetup.DreddLogs.Tests/Tests.cs

```diff
@@ -0,0 +1,54 @@
+﻿using System.Xml;
+using FluentAssertions;
+using FluentAssertions.Execution;
+using NUnit.Engine;
+using NUnit.Framework;
+
+namespace NUnit.OneTimeSetup.DreddLogs.Tests
+{
+    [TestFixture]
+    public class Tests
+    {
+        private ITestRunner _testRunner;
+
+        [OneTimeSetUp]
+        public void OneTimeSetup()
+        {
+            var testPackage = new TestPackage($"NUnit.OneTimeSetup.DreddLogs.Tests.Internal.dll");
+            _testRunner = TestEngineActivator.CreateInstance().GetRunner(testPackage);
+        }
+
+        [Test]
+        public void SimpleTest()
+        {
+            var testCaseCount = _testRunner.CountTestCases(TestFilter.Empty);
+
+            var xmlReport = _testRunner.Run(null, TestFilter.Empty);
+            var testCaseNodes = xmlReport.SelectNodes("//test-case");
+
+            using (new AssertionScope())
+            {
+                testCaseNodes.Count.Should().Be(testCaseCount);
+
+                foreach (var testCaseNode in testCaseNodes)
+                {
+                    var xmlNode = testCaseNode as XmlNode;
+
+                    var isFailed = xmlNode.SelectSingleNode("./failure") != null;
+                    isFailed.Should().BeTrue();
+
+                    var message = xmlNode.SelectSingleNode("./failure/message").InnerText;
+                    message.Should().Contain("NUnit.OneTimeSetup.DreddLogs.Exceptions.FixtureSetupException : Exception was thrown in fixture setup");
```

Comment:
You can use here useful FluentAssertion capability:
message.Should().Contain("text1")
              .And.Contain("text2);

Reply:
Okay, thanks for hint